tools/snp.sh

```diff
@@ -664,17 +711,21 @@ build_and_install_amdsev() {
   git remote set-url current "${AMDSEV_URL}"
   git fetch current "${amdsev_branch}"
   git checkout "current/${amdsev_branch}"
-
+   
+  if [ "$SVSM" = true ]; then
   # Based on latest AMDSEV documentation
   # Delete the ovmf/ directory prior to the build step for ovmf re-initialization
   [ ! -d "ovmf" ] || rm -rf "ovmf"
+  fi
```

Comment:
What is the purpose of this being wrapped in a conditional for SVSM only?

Reply:
@ramagali24 I'm thinking you mean to do this if SVSM flag was not passed? 

I think you have your conditionals mixed up

Reply:
Your reasoning makes sense. Since SVSM is currently using an older OVMF build, removing the directory for a fresh build can potentially cause issues, especially if SVSM has dependencies on its own specific OVMF structure or build process. It's a good idea to address this later when both SNP and SVSM converge on the same OVMF version from upstream sources.

For now, to avoid disrupting the SVSM build, we can skip removing the directory when SVSM is involved.

Reply:
No. You can't skip this for speed sake. This does nothing to break any level of the build process. It literally is centered around a conditional change. You need to address it and fix it.